src-cpp/rdkafkacpp.h

```diff
@@ -365,6 +365,21 @@ class RD_EXPORT DeliveryReportCb {
 
 
 /**
+ * @brief SSL Context Configuration callback class
+ *
+ */
+class RD_EXPORT SSLContextCb {
+ public:
+ /**
+  * @breif SSL context configuration callback
```

Comment:
"breif" typo.

This class needs to be fully documented (copying the blurb from rdkafka.h).